thoth-export-server/src/xml/doideposit_crossref.rs

```diff
@@ -1459,6 +1931,66 @@ mod tests {
         assert!(!output.contains(r#"        <doi>10.00001/PART.0001</doi>"#));
         assert!(!output.contains(r#"        <resource>https://www.book.com/part_one</resource>"#));
 
+        // Test basic Crossmark output for Active work
+        test_work.imprint.crossmark_doi =
+            Some(Doi::from_str("https://doi.org/10.00001/crossmark_policy").unwrap());
+        let output = generate_test_output(true, &test_work);
+        assert!(output.contains(r#"  <crossmark>"#));
```

Comment:
You're correct about the spacing (and I'll fix it), but weirdly, it's been passing tests all along when I run `cargo test --workspace -vv`... shouldn't it be failing these tests if the spacing is incorrect?

Reply:
OK, I figured out what's going on... a test will fail if a test string has _too many_ spaces, but not if it has _too few_. For example:

`assert!(output.contains(r#"      <crossmark>"#));`

If I add extraneous spaces, it will throw an error, because the string doesn't match. However, if I remove spaces, the test will pass, e.g. 

`assert!(output.contains(r#"  <crossmark>"#));`

Because `"  <crossmark>"` is contained within `"      <crossmark>"`

Reply:
I think this is unfortunately a necessary limitation of testing for this export; there's a comment in the file:

"    // Testing note: XML nodes cannot be guaranteed to be output in the same order every time
    // We therefore rely on `assert!(contains)` rather than `assert_eq!`"

I will fix all the spacing issues in the tests nonetheless in case we come up for a solution for this in the future :)